### DISCUSSION TITLE
New link to the Slack Workspace

### DISCUSSION THREAD

**Discussion by Bird87ZA** [goadesign/goa#2898] [General]
The link on the goa homepage is dead. I get an error saying that I should contact the person who sent me a link.

Is it possible to get a new link for this?

**Comment by raphael** [comment]
Hello, which link does not work for you? Link in README seems to be working for me.

**Comment by Bird87ZA** [comment]
Link in the readme takes me to this page:

![image](https://user-images.githubusercontent.com/1457444/137343498-e0760399-3e1b-4246-a7ae-f4019559d5e5.png)

When I try to sign in with Google, I get an error:

![image](https://user-images.githubusercontent.com/1457444/137343688-ec493665-73d8-406f-861c-32239a0124bf.png)

When I try to use the invite link at the bottom of https://goa.design (https://gophersinvite.herokuapp.com/) I get this:

![image](https://user-images.githubusercontent.com/1457444/137343963-bcd9f511-d5f7-4c67-95fd-5c0d87ce2af2.png)

**Comment by raphael** [comment]
I see, this link seems to work for signing up: https://invite.slack.golangbridge.org/ - I'll update https://goa.design.
